### PR TITLE
setup ray ml notebook based on the notebook format

### PR DESCRIPTION
**Pull request by harshad16**
setup ray ml notebook based on the notebook format
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: https://github.com/thoth-station/core/issues/311

## Description

Extracted the packages from: https://github.com/erikerlandson/ray-ubi/tree/main/images/ray-ml-notebook

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -18,10 +18,8 @@ managers:
   - name: version
     configuration:
       maintainers:
-        - fridex
-        - goern
         - harshad16
-        - pacospace
+        - erikerlandson
```
Comment (codificat) [inline .thoth.yaml:22]: As there is an OWNERS file, and kebechet takes mainainers from it if present, maybe just remove the `maintainers` section?

Although I see the proposed lists differ (maintainers in thoth.yaml vs approvers in owners). Is this intentional?

Comment (harshad16) [reply .thoth.yaml]: I have updated the approver's list.

**Comment by harshad16** [comment]
this pr is able to build an image successfully
https://quay.io/repository/thoth-station/s2i-ray-ml-notebook?tab=tags

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# ray core deps
```
Comment (erikerlandson) [inline requirements.txt:1]: do we want both a pipfile and a requirements.txt ?

Comment (harshad16) [reply requirements.txt]: no , we dont need both.
i have removed req.txt

**Pull request merged**